CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - `go.opentelemetry.io/otel/sdk/log/logtest` is now a separate Go module. (#6466)
 - `Recorder` in `go.opentelemetry.io/otel/log/logtest` no longer separately stores records emitted by loggers with the same instrumentation scope. (#6507)
 - Improve performance of `BatchProcessor` in `go.opentelemetry.io/otel/sdk/log` by not exporting when exporter cannot accept more. (#6569, #6641)
+- Updated documentation in `go.opentelemetry.io/otel/log`; no longer discourages direct usage of the Logs API. (#6675)
+- Renamed `Logs Bridge API` to `Logs API` across several documentation files for consistency. (#6675)
```

Comment:
```suggestion
- No longer discourage direct use of the `go.opentelemetry.io/otel/log` API. (#6675)
```

Reply:
Isn't it more like a fix (leftover) after #6167? @dmathieu, WDYT?

Reply:
Ah right, there's already a changelog entry. My bad then, no need for a second one.